web/genericcom.js

```diff
@@ -49,6 +51,25 @@ class GenericExternalServices extends DefaultExternalServices {
   static createL10n({ locale = "en-US" }) {
     return new GenericL10n(locale);
   }
+
+  static get scripting() {
+    const promise = loadScript(AppOptions.get("scriptingSrc")).then(() => {
+      return window.pdfjsSandbox.QuickJSSandbox();
+    });
+    const sandbox = {
```

Comment:
Is there a benefit to having all of these methods using `promise.then` vs have `scripting` return a promise and then have all the methods be direct calls? e.g.
```
get scripting() {
  return loadScript(...).then((sbx) => {
    return {
      createSandbox(data) {
         sbx.create(data);
      },
...
```

Reply:
It's for consistency with:
https://github.com/mozilla/pdf.js/blob/d3936ac9d2f7d11a8eeca6bdf86c3cd77cfbf02f/web/firefoxcom.js#L257-L269